lib/models/addon.js

```diff
@@ -412,16 +412,16 @@ var Addon = CoreObject.extend({
 
     @public
     @method included
-    @param {EmberApp} app The application object
+    @param {EmberApp|Addon} parent The parent object that included this addon
```

Comment:
I understand your concern about the param definition. Though as you can guess, I'm in favour of verbosity and clarity on this one.